doc/nrf/ug_bt_mesh_node_removal.rst

```diff
@@ -0,0 +1,205 @@
+.. _ug_bt_mesh_node_removal:
+
+Removing a node from a mesh network
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+In certain circumstances, a need may arise to remove a node from a Bluetooth® mesh network.
+This can be done by unprovisioning the node using the Config Node Reset message.
+
+Though the unprovisioning of the node is a straightforward procedure, some additional steps need to be taken to maintain the security of the network.
+The procedure consists of two steps:
+
+1. Removing the node.
+2. Replacing the network key(s), used by the removed node, in the rest of the nodes in the network.
+
+Removing the node
+*****************
+
+To remove the node from the network and reset its configuration, send the Config Node Reset message from a node running the Configuration Client model.
+The node running the Configuration Client model is referred to as the Mesh Network Manager.
+
+To remove the node using the |NCS| API, complete the following steps:
+
+1. Call the :c:func:`bt_mesh_cfg_cli_node_reset` function to send a Config Node Reset message to the node to be reset.
+#. Find the node in the Configuration Database (CDB) using the :c:func:`bt_mesh_cdb_node_get` function.
+#. Remove the node from the CDB using the :c:func:`bt_mesh_cdb_node_del` function.
+
+When using the nRF Mesh mobile app as a provisioner, do the following:
+
+1. On the Network screen, tap the mesh device node to be reset.
+#. Scroll down and find the :guilabel:`Reset Node` section.
+#. Press the :guilabel:`RESET` button.
+
+Updating the network key
+************************
+
+Even if the Config Node Reset message has been sent to the node, and the Config Node Reset Status message was received, this doesn't guarantee that the node has removed all configuration data.
+To ensure that the removed node is not able to decrypt network messages after being reset, update the network key(s) that were known to the removed node.
+
+This procedure can be used in any other cases when a network key needs to be updated.
+For example, in cases where there is a suspicion of network credentials being compromised.
+
+The same procedure can be used to update an application key.
+To update an application key, the associated network key must also be started before updating the application key.
```

Comment:
It reads bit strange. Something is missing?
```suggestion
To update an application key, the associated network key update must also be started before starting the update for the application key.
```

Reply:
I think I copied your suggestion from here: https://github.com/nrfconnect/sdk-nrf/pull/9056#discussion_r1011972485